pkg/traceql/combine.go

```diff
@@ -335,19 +338,25 @@ func QueryRangeCombinerFor(req *tempopb.QueryRangeRequest, mode AggregateMode) (
 	}
 
 	return &QueryRangeCombiner{
-		req:     req,
-		eval:    eval,
-		metrics: &tempopb.SearchMetrics{},
+		req:       req,
+		eval:      eval,
```

Comment:
Combiner is also using an evaluator. Is it possible to push the limiting logic to only exist in the evaluator and have calling code just ask it to see if it limited or not?

cc @mdisibio